### PR TITLE
Feat : Added watchLater

### PR DESCRIPTION
**Pull request by RahulSingh9131**
This branch contains
1. context and reducer for watch later
2. styles for components
3. & routes

### CODE REVIEW

File: src/css/screens/watchcard.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.watchcard-text h4{
+    margin-bottom: 5px;
+    font-size: var(--normal-font-size);
+}
+.watchcard-text p{
+    color: var(--text-color-light);
+    font-size: var(--small-font-size);
+}
+.watchcard-avatar{
+    height: 30px !important;
```
Comment (rohit8686) [inline src/css/screens/watchcard.css:30]: try to avoid using important by increasing specificity

**Review by rohit8686 — APPROVED** [review]
- Clean and modular code and nice class names 
- Looks good to merge

File: src/css/screens/watchcard.css (hunk 1 above)
Comment (RahulSingh9131) [reply src/css/screens/watchcard.css:30]: sure

**Review by Priyanshi-Tiwari — APPROVED** [review]
Looks good. The code is clean and descriptive.

**Pull request merged**